sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/DeploymentController.java

```diff
@@ -52,62 +54,109 @@ AiDeploymentDeletionResponse createAndDeleteDeploymentByConfigId(
   }
 
   /**
-   * Stop all deployments with the Java specific configuration ID
+   * Create and delete a deployment with the Java specific configuration ID.
+   *
+   * @param configId The configuration id.
+   * @param accept The accept header.
+   * @return a response entity with a string representation of the deployment creation response
+   */
+  @GetMapping("/by-config/{id}/createDelete")
+  ResponseEntity<Object> createAndDeleteDeploymentByConfigId(
+      @Nonnull @PathVariable("id") final String configId,
+      @RequestHeader(value = "accept", required = false) final String accept) {
+    final var response = createAndDeleteDeploymentByConfigId(configId);
+    if ("application/json".equals(accept)) {
+      return ResponseEntity.ok().body(response);
+    }
+    return ResponseEntity.ok("Deployment created and will be deleted.");
+  }
+
+  /**
+   * Stop all deployments with the Java specific configuration ID.
    *
    * <p>Only RUNNING deployments can be STOPPED
    *
    * @param configId The configuration id.
-   * @return the deployment modification response
+   * @param accept The accept header.
+   * @return a response entity with a string representation of the deployment modification response
    */
   @GetMapping("/by-config/{id}/stop")
   @Nonnull
-  @SuppressWarnings("unused") // debug method that doesn't need to be tested
-  List<AiDeploymentModificationResponse> stopByConfigId(
-      @Nonnull @PathVariable("id") final String configId) {
+  ResponseEntity<Object> stopByConfigId(
+      @Nonnull @PathVariable("id") final String configId,
+      @RequestHeader(value = "accept", required = false) final String accept) {
```

Comment:
(MInor)

```suggestion
      @Nullable @RequestHeader(value = "accept", required = false) final String accept) {
```

Applies to other methods as well.